### PR TITLE
Add support in HaveYouStopped RO command in command handler

### PR DESCRIPTION
**Pull request by yontyon**
Add Apollo support in Replica Specific Info. 
Specially, we add support in "Have you stopped" read-only command to support more reconfiguration tests.
Also, we have added a simple test in the reconfiguration tests to test the replica specific information feature in Apollo.

### CODE REVIEW

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 1]
```diff
@@ -122,6 +122,41 @@ async def test_wedge_command_with_state_transfer(self, bft_network):
 
         await self.validate_stop_on_super_stable_checkpoint(bft_network, skvbc)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 4 and c == 0)
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc_reconfiguration.py:126]: I'd suggest `selected_configs=lambda n, f, c: n == 7` instead

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 2]
```diff
@@ -122,6 +122,41 @@ async def test_wedge_command_with_state_transfer(self, bft_network):
...
+                while True:
+                    with trio.move_on_after(seconds=1):
+                        checkpoint_after = await bft_network.wait_for_checkpoint(replica_id=replica_id)
+                        if checkpoint_after == checkpoint_before + 2:
+                            break
+
+        await self.validate_stop_on_super_stable_checkpoint(bft_network, skvbc)
+
+        msg = skvbc.get_have_you_stopped_req()
+        rep = await client.read(msg, specific_replica_info=True)
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc_reconfiguration.py:156]: all_replicas ?

File: util/pyclient/bft_client.py [hunk 3]
```diff
@@ -167,23 +175,56 @@ async def send_loop(self, data, read_only):
                 self.reset_on_retry()
         return self.reply
 
+    async def send_sri_loop(self, data, replicas, num_of_replies=None):
```
Comment (robem) [inline util/pyclient/bft_client.py:178]: `sri` vs `rsi` Is there a difference? If not, please use one abbreviation only.

File: util/pyclient/bft_client.py [hunk 4]
```diff
@@ -72,6 +72,7 @@ def __init__(self, config, replicas):
         self.replies = dict()
         self.primary = None
         self.reply = None
+        self.rsi_reply = None
```
Comment (robem) [inline util/pyclient/bft_client.py:75]: The PR makes it clear but would you mind adding a comment that unfolds `rsi`

File: util/pyclient/bft_client.py [hunk 5]
```diff
@@ -199,8 +240,10 @@ async def recv(self, cancel_scope):
                 # coroutines. self.reply will get set in self.has_quorum()
                 cancel_scope.cancel()
 
-    def valid_reply(self, header):
+    def valid_reply(self, header, sri=False):
```
Comment (robem) [inline util/pyclient/bft_client.py:243]: `sri` same as above

**Pull request closed without merging**